cis194/week5/tien/src/Calc.hs

```diff
@@ -0,0 +1,74 @@
+{-# LANGUAGE TypeSynonymInstances #-}
+{-# OPTIONS_GHC -Wall #-}
+{-# LANGUAGE ViewPatterns         #-}
+
+module Calc where
+
+import           ExprT
+import           Parser
+import           StackVM
+
+-- Exercise 1 --
+eval :: ExprT -> Integer
+eval (Lit a)         = a
+eval (ExprT.Add a b) = eval a + eval b
+eval (ExprT.Mul a b) = eval a * eval b
```

Comment:
Instead of using `ExprT.Add`, we can import `ExprT` qualified. 
As in, `import qualified ExprT as ET` then, we are forced to use `ET.Mul`.
This is usually how we does it in production code base so that people know where the functions are coming from